### DISCUSSION TITLE
PowerApps FNU LNU's Environment

### DISCUSSION THREAD

**Discussion by briankim1027** [fusiondevkr/pl900#21] [1 Q&A]
본인이 생성한 개발자 계정 외에 제목에 쓴 것과 같은 FNU LNU 환경이 있는게 정상적인 상황인거죠?

**Comment by justinyoo** [comment]
맞습니다. `뭐라뭐라뭐라 (default)` 환경이 M365 개발자 계정을 생성하면 기본으로 생기구요, 파워 앱 개발자 계정을 생성하면 `아무개 환경` 이라고 하나 더 생겨욥.

**Reply by sesme100** [reply]
Power Apps 개발자 계정 = FNU LNU's Environment 로 생성됩니다. 

아무튼 감사드립니다.
